src/help/zaphelp/contents/releases/2_7_0.html

```diff
@@ -52,6 +52,9 @@
 <H3>VIEW core / urls</H3>
 Added optional <code>baseurl</code> parameter, to filter the URLs that are returned.
 
+<H3>VIEW ascan / scanProgress</H3>
+Returns the number of alerts raised by each scanner.
```

Comment:
Doesnt it return a lot more info than just this? The comment implies it _just_ returns the number of alerts :)